App.js

```diff
@@ -1,48 +1,114 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, Image, Fonts, Button } from 'react-native';
+import { Font } from 'expo';
 
 export default class App extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble-shadow': require('./assets/fonts/Double_Bubble_shadow.otf'),
+    });
+    this.setState({ fontLoaded: true });
+  }
   render() {
     return (
       <View style={styles.container}>
-        <View style={{flexDirection: 'row', paddingTop: 30}}>
-          <View>
-            <View style={styles.pongRoundedRectangle}>
-              <Text style={styles.pongMatesText}>Pong</Text>
+        <View style = {{flexDirection: 'row', justifyContent: 'space-around'}}>
```

Comment:
Put spaces between curly brackets and content: `{{ flexDirection: 'row', justifyContent: 'space-around' }}`